### DISCUSSION TITLE
Where is suitesparse used exactly?

### DISCUSSION THREAD

**Discussion by dawood95** [facebookresearch/theseus#433] [Q&A] [answered]
## ❓ Questions and Help

I couldn't find any specific file using suitesparse. Could someone point me in the right direction? thanks

**Comment by mhmukadam** [comment, marked as answer]
Hi @dawood95, we use suitesparse (scikit-sparse) in our [CholmodSparseSolver](https://github.com/facebookresearch/theseus/blob/main/theseus/optimizer/linear/cholmod_sparse_solver.py#L13-L14).
